package.json

```diff
@@ -149,6 +149,7 @@
         "webpack-dev-server": "^4.9.0",
         "yaml-flat-loader": "^1.0.1"
     },
+
```

Comment:
This change isn't needed.

Reply:
it is due to changes to fix the pr commit